### ISSUE TITLE
Equilibrium 

### ISSUE THREAD

**Issue by markstickley** [kinduff/dog-api#12]
Equilibrium is not being kept! The readme indicates that this repo is to keep the cat/dog equilibrium in check but I’m afraid while the APIs are not interoperable there is a certain unbalanced-ness to the whole situation! I’d really like to be able to use cats / dogs apis to distinguish between two environment with different settings without writing unique code for each api. Maybe adding a /fact endpoint and a /breeds endpoint would do it…
Or perhaps we (developers, the internet, the world and its people) need a generic pets api that would allow for simple extension to accommodate other species…

**Comment by kinduff** [comment]
I agree with you. I'm currently in the process of release V2 for this API in a whole different architecture, including breeds, pictures, etc. But I didn't thought about doing a Pet API for the matter.

Maybe it's a good opportunity to at least standardize the API responses (JSON API, could be), in order to have similar APIs for different pets?

**Comment by markstickley** [comment]
Sounds great! Definitely worth at least considering it while working on v2 in case there is time to expand it further :)
